spec/fixtures/notification_layout/create_template.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "1",
```

Comment:
Why this answer is not structure like the rest of the other answers? @thebluber  @aliaksandr-volkau 
@danielvysotskyi Is it normal or it has been overlooked? 

There is no `data` key in the answer